### PR TITLE
*: remove errors.Trace in main handle path

### PR DESCRIPTION
**Pull request by lysu**
### What problem does this PR solve? 

remove errors.Trace call in some high-level package(which is safe to remove)

- session
- planner 
- executor
- tables 
- kv

more sophisticated like tikvclient or types will be handle later

### What is changed and how it works?

remove error.Trace if err has be traced.

### Check List 

Tests 

 - Unit test
 - Integration test

Code changes

 - many modification

Side effects

 - Miss error stack if wrong remove

Related changes

 - N/A

---
This change is [Reviewable](https://reviewable.io/reviews/pingcap/tidb/10033)

### CODE REVIEW

File: distsql/select_result.go [hunk 1]
```diff
@@ -15,6 +15,7 @@ package distsql
 
 import (
 	"context"
+
```
Comment (jackysp) [inline distsql/select_result.go]: remove this empty line?

File: domain/info.go [hunk 2]
```diff
@@ -17,6 +17,7 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+
```
Comment (jackysp) [inline domain/info.go]: ditto

File: executor/adapter.go [hunk 3]
```diff
@@ -16,6 +16,7 @@ package executor
 import (
 	"context"
 	"fmt"
+
```
Comment (jackysp) [inline executor/adapter.go]: ditto

File: executor/admin.go [hunk 4]
```diff
@@ -15,9 +15,9 @@ package executor
 
 import (
 	"context"
+
```
Comment (jackysp) [inline executor/admin.go]: ditto

File: executor/aggregate.go [hunk 5]
```diff
@@ -15,6 +15,7 @@ package executor
 
 import (
 	"context"
+
```
Comment (jackysp) [inline executor/aggregate.go]: ditto

File: executor/analyze.go [hunk 6]
```diff
@@ -15,6 +15,7 @@ package executor
 
 import (
 	"context"
+
```
Comment (jackysp) [inline executor/analyze.go]: ditto

File: executor/builder.go [hunk 7]
```diff
@@ -16,6 +16,7 @@ package executor
 import (
 	"bytes"
 	"context"
+
```
Comment (jackysp) [inline executor/builder.go]: ditto

File: executor/checksum.go [hunk 8]
```diff
@@ -15,9 +15,9 @@ package executor
 
 import (
 	"context"
+
```
Comment (jackysp) [inline executor/checksum.go]: ditto

File: executor/ddl.go [hunk 9]
```diff
@@ -16,6 +16,7 @@ package executor
 import (
 	"context"
 	"fmt"
+
```
Comment (jackysp) [inline executor/ddl.go]: ditto

File: distsql/request_builder.go [hunk 10]
```diff
@@ -38,7 +38,10 @@ type RequestBuilder struct {
 
 // Build builds a "kv.Request".
 func (builder *RequestBuilder) Build() (*kv.Request, error) {
-	return &builder.Request, errors.Trace(builder.err)
+	if builder.err != nil {
+		return &builder.Request, errors.Trace(builder.err)
```
Comment (zz-jason) [inline distsql/request_builder.go]: no need to trace

**Pull request merged**